Makefile

```diff
@@ -140,12 +140,17 @@ help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 ##@ Development
-
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) \
-		paths=./internal/controllers \
-		rbac:roleName=manager-role
+manifests: vendor controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./..." output:crd:artifacts:config=hack/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./vendor/sigs.k8s.io/cluster-api/..." output:crd:artifacts:config=hack/crd/bases
```

Comment:
I'm not a fan of using vendor if it's possible to avoid it, generating CRDs in vendor also adds some complexity to development

Reply:
+1, vendoring bloats the repo a lot from my experience and technically should not be needed, at least for now IMO

Reply:
It is being ignored by the gitignore, so it is not being used in the build process and won’t bloat the repo. Also this is not a task you would manually run for doing any commands, it is only triggered by generate tasks. Only thing it does here, is providing a relative directory for the CRDs to be picked up from. This is workflow wich worked perfectly for me in the past and allows us to avoid manually writing boilerplate code where possible. Additionally, when (if) we will be able to import rancher cluster from the rancher repo, there will be no business logic changes thanks to this refactor and centralized use of client.Object interface compatible structures.

Reply:
As its ignored are we happy to resolve this?